bin/run-tc-ru_transfer.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -xe
+
+ru_dir="./data/ru"
+ru_csv="${ru_dir}/ru.csv"
+
+epoch_count=$1
+
+if [ ! -f "${ru_dir}/ru.csv" ]; then
+    echo "Downloading and preprocessing Russian example data, saving in ${ru_dir}."
+    python -u bin/import_ru.py ${ru_dir}
+fi;
+
+# Force only one visible device because we have a single-sample dataset
+# and when trying to run on multiple devices (like GPUs), this will break
+export CUDA_VISIBLE_DEVICES=0
+
+# WORKS: #
+
+echo "# CUDNN FINE"
+python -u DeepSpeech.py --noshow_progressbar --noearly_stop \
+       --fine_tune \
+       --alphabet_config_path "${ru_dir}/alphabet.ru" \
+       --load "transfer" --drop_source_layers 1 \
+       --train_files  "${ru_dir}/ru.csv" --train_batch_size 1  \
+       --dev_files  "${ru_dir}/ru.csv" --dev_batch_size 1 \
+       --test_files  "${ru_dir}/ru.csv" --test_batch_size 1 \
+       --checkpoint_dir '/tmp/ckpt/transfer' --epochs 5 \
```

Comment:
I see you use the same checkpoint directory everywhere, is that fine ? Do we want to ensure a bit more about the behavior of the produced model ?

Reply:
please elaborate... what do we want to ensure?

Reply:
Like, checking what does that models returns for a specific WAV file, so we ensure it's not regressing at inference level. Would it make sense ?

Reply:
Without a language model + trie, the output is gibberish, so this is not possible at the moment. I think downloading a language model every task-cluster run is overkill